openlibrary/plugins/openlibrary/js/wikipedia_citation_button.js

```diff
@@ -0,0 +1,5 @@
+export function WikipediaCitationButtonPressed() {
+    $('#wikilink').click(function() {
```

Comment:
This doesn't match the inline script.
The inline script sets a focus, it doesn't set the focus on click.

Reply:
Fixed!